syntax = "proto3";
import "common.proto";
//import "google/protobuf/empty.proto";
package gs2ms;
option cc_generic_services = true;


message StartGSRequest
{
  IpPort rpc_client = 1;
  IpPort rpc_server = 2;
  uint32 node_id = 3;
  uint32 server_type = 4;
  uint64 master_server_addr = 5;
}

message StartGSResponse
{
  uint32 master_node_id = 1;
  repeated SceneInfo scenes_info = 2;
  uint32 gateway_node_id = 3;
}

service G2msService {
  rpc StartGS (StartGSRequest) returns (StartGSResponse);
}
